pkgs/applications/video/epgstation/update.nix

```diff
@@ -14,8 +14,8 @@ let
     attrPath = lib.toLower pname;
     rev-prefix = "v";
   };
-  updateScript = builtins.elemAt updater 0;
```

Comment:
Might I interest you in [`_experimental-update-script-combinators.sequence`](https://github.com/NixOS/nixpkgs/blob/6910e5840f961621b440a3fcf57d503a96d5e645/pkgs/common-updater/combinators.nix#L110-L113)?

Reply:
I am not the maintainer of the update-mirakurun and update-epgstation scripts. Therefore I will change only the minimum necessary now.